docs/daemon/snapshot.md

```diff
@@ -38,3 +38,19 @@ Snapshot current node's latest data
 ```bash
  iris snapshot --home=<path-to-your-home>
 ```
+
+## FAQ
+
+### What happens if everyone delete previous stored blockchain data and a new machine will try to sync up
+
+If that happens, a new machine will be unable to sync from scratch, but it can use a snapshot to catch up faster.
+
+I think some of the nodes will not delete all the previous data, such as nodes for explorers and wallets. And we, the IRIS Foundation will keep all the data too, also we can offer a full data snapshot. And we encourage snapshot service providers could provide both the minimal and the full snapshot :)
```

Comment:
we should avoid using `I`. (just I think)

Reply:
Agreed.  "Presumably not all nodes will delete historical data, ..."